### ISSUE TITLE
xclip - No such file or directory

### ISSUE THREAD

**Issue by bodryj** [dshoreman/nextshot#72]
```
$ nextshot -ac
Waiting for selection...
Copying image to clipboard...
xclip: -t: No such file or directory

Aborted due to error
```
Manjaro KDE

what could be the problem?

**Comment by dshoreman** [comment]
Thanks for the report!

After reproducing locally, it looks like this bug was introduced with the support for setting image type. I've managed to pinpoint the issue so if I have time later tonight I'll see if I can get a hotfix released.

**Comment by dshoreman** [comment]
Fixed in v1.4.1, thanks again!

**Comment by bodryj** [comment]
Thanks! All work nextshot 1.4.1-1
